### DISCUSSION TITLE
Could Someone show me how I can use this tool to get my .flp file to load VST2 instead of VST3 for a specific plugin?

### DISCUSSION THREAD

**Discussion by CamObi-san** [demberto/PyFLP#194] [Q&A]
I'm having an issue with a specific plugin not recalling its settings properly with a vst plugin for which this is a known bug with the vst3 version of the plugin but not the vst2 version. I'm hoping to edit the .fp file to tell it to load the vst2 version of that plugin instead of the vst3 version in order to see if that will fix my issue. I have Python 3.10 and 3.8 installed and also have download pypy. I am using Windows 10.

**Comment by NickDangerous** [comment]
You can simply delete the vst3 file, I think.  FL will load the VST2 version instead.
